maestro/audit.py

```diff
@@ -56,6 +57,12 @@ def _format_what_compact(self, what):
         return '{} container{}'.format(
             len(what), 's' if len(what) > 1 else '')
 
+    def _format_what_services_ships(self, what):
+        if isinstance(what, entities.Entity):
+            return None
+        return(','.join(map(lambda e: '{}_{}'.format(e.service.name,
```

Comment:
After discussing with you, found a better way (i hope!) :-)  Let me know if i need to document this!